src/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Source files
+file(GLOB_RECURSE SOURCES_SC2LADDERSERVER "*.cpp" "*.h")
+
+# Include directories
+include_directories(SYSTEM
+    ${PROJECT_SOURCE_DIR}/s2client-api/include
+    ${PROJECT_SOURCE_DIR}/s2client-api/contrib/protobuf/src
+    ${PROJECT_BINARY_DIR}/s2client-api/generated
+    ${PROJECT_SOURCE_DIR}/rapidjson
+    ${PROJECT_SOURCE_DIR}/curl/include/curl
```

Comment:
Probably you should hide curl headers under 'if' too.

Reply:
Added